root/static/scripts/timeline.js

```diff
@@ -111,11 +111,13 @@ class TimelineViewModel {
         }, 1000);
 
         self.waitToGraph = ko.computed(function () {
-            if (_.some(self.enabledCategories(), function (c) { return c.hasLoadingLines() }))
+            if (_.some(self.enabledCategories(), function (c) { return c.hasLoadingLines() })) {
                 return true;
+            }
 
-            if (self.options.events() && !self.loadedEvents())
+            if (self.options.events() && !self.loadedEvents()){
```

Comment:
We're missing a space before the `{` here:
```suggestion
            if (self.options.events() && !self.loadedEvents()) {
```